src/caret_analyze/record/record.py

```diff
@@ -309,10 +309,19 @@ def _to_dataframe(
         # so here uses a dictionary type.
         df_dict: Dict[str, List[Optional[int]]]
         df_dict = {c: [None]*len(df_list) for c in columns}
+
+        int64_max = 2**63-1
+
         for i, df_row in enumerate(df_list):
             for c in columns:
                 if c in df_row:
-                    df_dict[c][i] = df_row[c]
+                    # uint64 to int64 conversion.
+                    # This is workaround to fix some uint64 trace points.
+                    if df_row[c] <= int64_max:
+                        df_dict[c][i] = df_row[c]
+                    else:
+                        # convert uint64 to int64
+                        df_dict[c][i] = ~(df_row[c] & int64_max)
```

Comment:
`df_dict[c][i]` becomes negative value `if df_row[c] > int64_max` . Is it acceptable? Or should we use fixed value if overflow happens, like `df_dict[c][i] = int64_max`

Reply:
Since you mentioned it's workaround and if the purpose of this PR is to cast uint64 to int64 anyway, I think the current implementation is okay.

Reply:
> Or should we use fixed value if overflow happens, like df_dict[c][i] = int64_max

I see. It looks also good to use fixed value.
However, current caret_analyze_cpp_impl cast int64_t to uint64_t (This is critical bug!).
Anyway, I'll summarize which trace points may face this problem fist.
With usual usage, converting should not called.
So, I'll add warning message to inform for users.

Reply:
Understood. Thank you!